testing/python/language/test_tilelang_language_nvf4_mma_block_scale.py

```
@@ -499,16 +701,44 @@ def test_sm120_fulltile_package_contract_supports_k128_path():
-    assert (contract.warp_issues, contract.warpgroup_issues) == (24, 96)
+    assert (contract.tile_m, contract.tile_n) == (256, 128)
+    assert (contract.sfa_words, contract.sfb_words) == (4, 2)
+    assert contract.words_per_stage == contract.kblocks
+
+    for lane in (0, 1, 2, 17, 31):
+        for warp_m in range(contract.block_row_warps):
+            for warp_n in range(contract.block_col_warps):
+                sfa_rows, sfb_rows = contract.compact_selector_scale_rows(lane, warp_m, warp_n)
+                for kblock in range(contract.kblocks):
+                    sfa_offsets, sfb_offsets = contract.compact_selector_scale_word_offsets(lane, warp_m, warp_n, kblock)
+                    for row, flat in zip(sfa_rows, sfa_offsets):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)